### ISSUE TITLE
[GTK] misaligned fancy background

### ISSUE THREAD

**Issue by sandsmark** [QtCurve/qtcurve#7]
![example](https://f.cloud.github.com/assets/25765/1754566/d7f0aa34-6653-11e3-9b68-d3f5d36a1ef3.png)

It seems like the margin or padding or something isn't taken into the account for painting the fancy background stuff.

Pretty low importance, but I thought I'd add it here for tracking anyways.

**Comment by yuyichao** [comment]
Could you explain the problem?

**Comment by sandsmark** [comment]
If you look at the (misused) progress bars, the fancy squares seem to be shifted to the right (by the same amount as they are indented).

**Comment by yuyichao** [comment]
I C ....
It will probably take some time before this is fixed because I haven't fully get into the gtk code yet...

**Comment by sandsmark** [comment]
Yeah, I looked at it and it wasn't entirely clear where the issue was (I found the code for drawing the pretty squares, but it looks like it is whatever calls the function that calls that function that should take the indentation into account).

And as I said, pretty low priority, I just thought I'd note it here before I forgot it. :-)
